sdk/src/main/java/com/horizen/utils/BytesUtils.java

```diff
@@ -190,4 +195,54 @@ public static boolean contains(Collection<byte[]> collection, byte[] value) {
         }
         return false;
     }
+
+    private static int HorizenPublicKeyAddressPrefixLength = 2;
+    private static int HorizenPublicKeyAddressHashLength = 20;
+    private static int HorizenPublicKeyAddressChecksumLength = 4;
+
+    private static int HorizenPublicKeyAddressBase58CheckLength = 35;
+
+    private static byte[] PublicKeyMainNetPrefix = BytesUtils.fromHexString("2089"); // "zn"
```

Comment:
the name shall starts from a lower case or the whole name shall be in CAMEL_CASE

Reply:
changed to UPPER_CASE as in other places.